src/transport.ts

```diff
@@ -24,4 +45,23 @@ export class WebRTCTransport implements Transport {
 	get [symbol](): true {
 		return true
 	}
+
+	todo_cb() {
+	}
+
+	_connect (ma: Multiaddr, options: WebRTCDialOptions) {
+		this.channel = this.peer_connection.createDataChannel("channel");
```

Comment:
This isn't part of the spec yet, but we have been using the label `"data"` here. Also, this channel should have `{ negotiated: true, id: 1 }` in the creation options, as this is negotiated as part of the SDP.